docker-compose.yml

```diff
@@ -51,28 +65,36 @@ services:
     build:
       context: ./worker
     environment:
+      # Limit RAM usage of each container because containers share a VM
+      - JVM_INITIAL_RAM=${JVM_INITIAL_RAM:-5.0}
+      - JVM_MIN_RAM=${JVM_MIN_RAM:-5.0}
+      - JVM_MAX_RAM=${JVM_MAX_RAM:-15.0}
+      # DB credentials
       - AB2D_DB_HOST=db
       - AB2D_DB_PORT=5432
       - AB2D_DB_DATABASE=ab2d
       - AB2D_DB_SSL_MODE=allow
       - AB2D_DB_USER=ab2d
       - AB2D_DB_PASSWORD=ab2d
+      # Location to store files within container
       - AB2D_EFS_MOUNT=/opt/ab2d
-      - AB2D_EXECUTION_ENV=local
+      # LOCAL if running on person machine, CI if running on Jenkins agent VM
+      - AB2D_EXECUTION_ENV=${AB2D_EXECUTION_ENV}
```

Comment:
This is used for EventLogger. local is the only value that turns off kinesis logging so the value of ci would cause a bunch of error messages.

Reply:
AB2D_EXECUTION_ENV, that is

Reply:
Addressed. Hard coded to local in docker-compose now. Testing in Jenkins and Travis